src/components/body/DocTool/SMEPortal/SMEPortalHome/SMEPortalHome.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import '../../../RoutesApp/RoutesApp.css';
+import aruba from '../../../../../assets/aruba_logo.png';
+import HorizontalCard from '../../../../modules/HorizontalCard/HorizontalCard'
```

Comment:
So you haven't used the custom card we made?

Reply:
Refactored the custom card name to VerticalCard and HorizontalCard, in `modules`